E.Deezer/ExecutorService.cs

```diff
@@ -218,39 +178,15 @@ private Stream GetDecompessionStreamForResponse(Stream responseStream, HttpConte
             return responseStream;
         }
 
-        private async Task<T> DeserializeResponseStream<T>(HttpResponseMessage response)
+        /// <summary>
+        /// Checks the response for HTTP errors and Exceptions.
+        /// </summary>
+        /// <exception cref="HttpRequestException">Occurs
+        /// when the response was not HTTP 200 or when the response
+        /// content is null.</exception>
+        private void Check(HttpResponseMessage response)
         {
-            using (var responseStream = await response.Content.ReadAsStreamAsync()
-                                                              .ConfigureAwait(false))
-            {
-                using (var compressedStream = GetDecompessionStreamForResponse(responseStream, response.Content.Headers))
-                {
-                    using (var reader = new StreamReader(compressedStream))
-                    {
-                        using (var jsonReader = new JsonTextReader(reader))
-                        {
-                            return this.jsonSerializer.Deserialize<T>(jsonReader);
-                        }
-                    }
-                }
-            }
-        }
```

Comment:
This deletion a bit misleading because the library still handle the compression like before:
https://github.com/projectgoav/E.Deezer/blob/ef47e238847a76b58b04773c64410d000e116592/E.Deezer/ExecutorService.cs#L59-L65
As You can see the `GetAsync` method will decompress the server response (`PostAsync` and `DeleteAsync` calls GetAsync).

Reply:
Oops! I totlaly missed that when reviewing this. Github's review pages aren't always the clearest. Sorry!